modules/prelude-org.el

```diff
@@ -51,6 +51,8 @@
     (define-key newmap (kbd "C-c -") nil)
     (define-key newmap (kbd "C-a") 'org-beginning-of-line)
     (make-local-variable 'minor-mode-overriding-map-alist)
+    ;; windmove bindings clash badly with org mode, so disable them in org mode buffers
+    (windmove-mode -1)
```

Comment:
That's a global minor mode, so you'll disable it everywhere, not just in org-mode with this approach.

Reply:
Ah, I overlooked that, sorry. 

So I think there's not a straightforward way to do this, but it will have to be done by manually overriding the keybindings that windmove has installed, by poking into its code and seeing what they are. That's a shame. 😢

Reply:
There are a few alternative suggestions how to solve this here https://www.reddit.com/r/orgmode/comments/kyj1zi/using_windmove_with_meta_and_orgmode_conflicts/, but indeed there's no very simple solution.

Reply:
There's also a solution suggested by org-mode's team here https://orgmode.org/manual/Conflicts.html